python/samples/04.ai.g.autogen.product-spec-critique/src/bot.py

```diff
@@ -0,0 +1,107 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+
+Description: initialize the app and listen for `message` activitys
+"""
+
+import os
+import sys
+import traceback
+from botbuilder.schema import Activity, ActivityTypes
+
+from botbuilder.core import TurnContext, MemoryStorage
+from teams import Application, ApplicationOptions, TeamsAdapter
+from teams.ai import AIOptions
+from teams.ai.actions import ActionTypes, ActionTurnContext
+from autogen_planner import AutoGenPlanner, PredictedSayCommandWithAttachments
+
+from config import Config
+from state import AppTurnState
+
+from spec_critique_group import SpecCritiqueGroup
+
+config = Config()
+def build_llm_config():
```

Comment:
please move config handling to a class in its own file like so https://github.com/microsoft/teams-ai/blob/main/python/samples/07.planners.a.langchain/src/config.py to match our other samples